app/admin/tenants/tenants.py

```diff
@@ -189,3 +198,25 @@ async def get_tenant(
     wallet = await aries_controller.multitenancy.get_wallet(wallet_id=tenant_id)
 
     return tenant_from_wallet_record(wallet)
+
+
+@router.get("/group/{group_id}", response_model=List[Tenant])
+async def get_tenants_by_group(
```

Comment:
The get wallets method when using the wallet gorups plugin now supports a group_id property. Can we merge the get_tenants and get_tenants_by_group and make the group_id a query param of the get_tenants call?

Reply:
@morrieinmaas this hasn't been resolved yet. I would like the get_tenants and get_tenants_by_group to be a single endpoint. 

We can just make the group id a query param (that is optional) to the get_tenants endpoint. Just like the endpoint in acapy

Reply:
ah sure